src/renderer/containers/QuickStart/VerifyEmailStep.tsx

```diff
@@ -76,9 +76,17 @@ export const VerifyEmailStep: React.FC = () => {
   const setEmailVerificationSent = useAccount((store) => store.setEmailVerificationSent);
 
   const handleCheckVerification = async () => {
-    authService.refreshUser().catch((err) => {
+    try {
+      await authService.refreshUser();
+
+      // Get current user manually since the user variable above hasn't updated yet
+      const newUser = authService.getCurrentUser();
+      if (!newUser?.emailVerified) {
+        showError("Email still not verified. Check spam folder?");
```

Comment:
```suggestion
        showError("Email is not yet verified. Have you checked your spam folder?");
```
I think I'd prefer a more fuller sentence here.

Reply:
I made it shorter to prevent the multi-line, hoping people would actually read it lol.

Reply:
```suggestion
        showError("Email still not verified. Checked spam folder?");
```
my only suggestion is to imply "[have you] checked [your] spam folder?"

or go with a more direct approach
```suggestion
        showError("Email still not verified. Check your spam folder");
```

Reply:
I just don't think adding an extra 5 words or so is going to be the deciding factor of whether people read the prompt or not.

Reply:
It doesn’t matter much to me. I think it’s far more important we have some kind of error and get it live quickly rather than deliberating on it.

Just change it to whatever and push it imo.